### ISSUE TITLE
Could not create the embedder backing store.

### ISSUE THREAD

**Issue by jtmuller5** [Dropsource/monarch#121]
I'm not exactly sure what's causing the issue but occasionally I'm getting into a state where the preview pane is stuck on one story. I can select other stories but the preview doesn't update. I also see this error in the console:

```
[ERROR:flutter/shell/platform/embedder/embedder.cc(1077)] Could not create the embedder backing store.
``` 

Here's my code for the story. I use get_it as a service locator so I'm trying to "set up" [get_it](https://pub.dev/packages/get_it) before the story runs:
```
Widget dentist_header_all_details() {
  if (!GetIt.instance.isRegistered<LastUsedService>()) {
    GetIt.instance.registerSingleton<LastUsedService>(LastUsedService());
  }

  if (!GetIt.instance.isRegistered<LocationService>()) {
    GetIt.instance.registerSingleton<LocationService>(LocationService());
  }

  if (!GetIt.instance.isRegistered<SharedPreferences>()) {
    GetIt.instance.registerSingleton<SharedPreferences>(MockSharedPreferences());
  }

  return DentistHeader(
    getTestDentist(),
    showLocation: true,
    showRating: true,
    showYearsInPractice: true,
    showInfoButton: true,
    showBuffer: true,
    showDistance: true,
  );
}
```

![image](https://github.com/Dropsource/monarch/assets/47997351/1c8fd950-c83b-4ad4-82da-9a042cd6e04b)

**Comment by jtmuller5** [comment]
Quick follow up, I get this error even without using get_it or trying to perform setup steps before the story.

I'm on a Mac with M1 chip if that helps.

**Comment by fertrig** [comment]
I'll be looking at this soon.

**Comment by fertrig** [comment]
I merged the fix into main branch. We will be running tests now. I'll re-open this PR until I release the fix.

**Comment by fertrig** [comment]
Fix released with monarch 2.2.2, see [release notes](https://monarchapp.io/blog/version-2.2.2).
